services/sensu_webhook.rb

```diff
@@ -0,0 +1,22 @@
+class SensuWebhook
+  REDIS_KEY_PREFIX = 'sensu'
+
+  def initialize(payload)
+    @payload = payload
+  end
+
+  def process
+    service_id = @payload['key']
+    redis_key = "#{REDIS_KEY_PREFIX}/#{service_id}"
```

Comment:
Can be, I just followed what I did for Pingdom and there I used `/`. But there isn't a particular reason, so I'll change both and update the PR.